src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Receivers.kt

```diff
@@ -199,4 +199,18 @@ class MarkdownTaskReceiver(
         val commandLine = substitutor.replace(rawCommandLine)
         return runCommandLine(commandLine)
     }
+
+    private fun compileMarkdown(request: CompileRequest, convertedMarkdown: File): CompileResponse {
```

Comment:
Come on. This method _does not_ compile markdown, it compiles tex. It also doesn't care where the tex file came from, was it converted from Markdown or anything else

Reply:
Done